public/js/controllers/traffic_graphs_controller.js

```diff
@@ -208,6 +208,42 @@ export class TrafficGraphsController {
             yLimitSentFiltered = d3.max(tenMinGraphFilteredData, d => d.total_sent),
             yLimitFailed = d3.max(dailyFailedTotal, d => d.total_errored),
             yLimitFailedFiltered = d3.max(tenMinGraphFilteredData, d => d.total_errored); 
+        
+        // Add Y axis Slider
+        total_received_sms_graph
+            .append("g")
+            .attr("class", "receivedSlider")
```

Comment:
the font colour for the slider values is quite faint. Lets make it more visible.

Reply:
The active/hovered value is clear.